en/brands.ftl

```diff
@@ -121,7 +121,10 @@
 
 ## Facebook products
 
+-brand-name-facebook-messenger = Facebook Messenger
```

Comment:
@peiying2 note the adddition of some new brand terms here. Since https://github.com/mozilla/bedrock/pull/9048 was added, bedrock will now fallback to `en` brand terms if a locale does not translate ithe term. So there should now no longer be a need to manually copy these to every brands.ftl locale file.